corehq/warehouse/transforms/sql/domain_dim.sql

```diff
@@ -0,0 +1,55 @@
+INSERT INTO {{ domain_dim }} (
+    domain_id,
+    domain,
+    default_timezone,
+    hr_name,
+    creating_user_id,
+    project_type,
+    deleted,
+    is_active,
+    case_sharing,
+    commtrack_enabled,
+    is_test,
+    location_restriction_for_users,
+    use_sql_backend,
+    first_domain_for_user,
+    domain_last_modified,
+    domain_created_on,
+    dim_last_modified,
+    dim_created_on
+)
+SELECT
+    domain_id,
+    domain,
+    default_timezone,
+    hr_name,
+    creating_user_id,
+    project_type,
+    CASE doc_type
+        WHEN 'Domain' THEN false
+        WHEN 'Domain-Deleted' THEN true
+    END,
+    is_active,
+    case_sharing,
+    commtrack_enabled,
+    CASE is_test
+        WHEN 'none' THEN false
+        WHEN 'true' THEN true
+        WHEN 'false' THEN false
+    END,
+    location_restriction_for_users,
+    -- Ensures all the boolean values are either true or false
+    CASE use_sql_backend
+        WHEN use_sql_backend THEN true
+        ELSE false
```

Comment:
could also be null. probably not necessary and i can make this field accept nulls